doc/api/errors.md

```diff
@@ -970,172 +962,166 @@ made to mark a stream and dependent of itself.
 <a id="ERR_HTTP2_UNSUPPORTED_PROTOCOL"></a>
 ### ERR_HTTP2_UNSUPPORTED_PROTOCOL
 
-Used when `http2.connect()` is passed a URL that uses any protocol other than
-`http:` or `https:`.
+`http2.connect()` was passed a URL that uses any protocol other than `http:` or
+`https:`.
 
 <a id="ERR_INDEX_OUT_OF_RANGE"></a>
 ### ERR_INDEX_OUT_OF_RANGE
 
-Used when a given index is out of the accepted range (e.g. negative offsets).
+A given index is out of the accepted range (e.g. negative offsets).
 
 <a id="ERR_INSPECTOR_ALREADY_CONNECTED"></a>
 ### ERR_INSPECTOR_ALREADY_CONNECTED
 
-When using the `inspector` module, the `ERR_INSPECTOR_ALREADY_CONNECTED` error
-code is used when an attempt is made to connect when the inspector is already
-connected.
+While using the `inspector` module, an attempt was made to connect when the
+inspector is already connected.
 
 <a id="ERR_INSPECTOR_CLOSED"></a>
 ### ERR_INSPECTOR_CLOSED
 
-When using the `inspector` module, the `ERR_INSPECTOR_CLOSED` error code is
-used when an attempt is made to use the inspector after the session has
-already closed.
+While using the `inspector` module, an attempt was made to use the inspector
+after the session has already closed.
 
 <a id="ERR_INSPECTOR_NOT_AVAILABLE"></a>
 ### ERR_INSPECTOR_NOT_AVAILABLE
 
-Used to identify when the `inspector` module is not available for use.
+The `inspector` module is not available for use.
```

Comment:
`s/is/was/`

Reply:
I think I'll keep present tense on this one because it describes something about the binary in the present. (I'm wondering if present tense would be best for all of these, but that would be a heck of a rewrite at this point. Might consider it for a larger doc change to get tense usage consistent everywhere though.)